integreat_cms/cms/views/events/event_form_view.py

```diff
@@ -116,6 +116,11 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
         recurrence_rule_form = RecurrenceRuleForm(
             instance=recurrence_rule_instance,
             disabled=disabled,
+            initial={
+                "recurrence_end_date": recurrence_rule_instance.recurrence_end_date
+                if recurrence_rule_instance
+                else None
+            },
```

Comment:
First I was confused whether this was valid at all, until I realized that this wasn't a dict comprehension. I think this would make it more readable
```suggestion
            initial={
                "recurrence_end_date": (
                    recurrence_rule_instance.recurrence_end_date
                    if recurrence_rule_instance
                    else None
                )
            },
```